### ISSUE TITLE
[Bug]: Performance of CN Primary Key Deduplication Significantly Inferior to DN Primary Key Deduplication

### ISSUE THREAD

**Issue by sukki37** [matrixorigin/matrixone#10427]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

Run tpcc 10-100 on 129:

<img width="1037" alt="企业微信截图_50ee249f-aac3-42cb-92c7-8830eb5686ac" src="https://github.com/matrixorigin/matrixone/assets/77312370/cc312459-5f94-4b42-985f-073dfcaec156">

### Expected Behavior

We expected the CN primary key deduplication to perform at least at a comparable level to the DN primary key deduplication, considering similar hardware and network conditions.

**Comment by fengttt** [comment]
Wait -- how can you get rid of insert check?

**Comment by ouyuanning** [comment]
rid 是指啥？

**Comment by ouyuanning** [comment]
rid 是指啥？

**Comment by ouyuanning** [comment]
还在处理

**Comment by ouyuanning** [comment]
去掉了pk查重中，点写的join节点后. tpcc能从110提升到180左右。
还是很慢。

**Comment by ouyuanning** [comment]
tpcc的性能基本一样了。

**Comment by sukki37** [comment]
@aressu1985  please help on verify

**Comment by aressu1985** [comment]
testing

**Comment by aressu1985** [comment]
fixed:935e4be378d6bac8e1e23d1d2b3e72c861d348b2
乐观+CN全量去重+DN增量去重
<img width="1338" alt="image" src="https://github.com/matrixorigin/matrixone/assets/47846308/8c1b3d10-3e66-43e5-9d5c-898c69c5cbda">
